core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/item/Aggregators.java

```diff
@@ -1048,6 +1048,31 @@ public SetConsumer<T> newItemConsumer(IType<IItem> itemType) {
 		return filter(distinctAsString(attribute, ", "), ItemFilters.type(typeId)); //$NON-NLS-1$
 	}
 
+	public static <T> IAggregator<Number, ?> getIdentifier(String typeId, IAttribute<T> attribute) {
+		IAggregator<Set<T>, ?> aggregator = Aggregators.distinct(attribute);
+		aggregator = filter(aggregator, ItemFilters.type(typeId));
+		return Aggregators.valueBuilderAggregator(aggregator, new IValueBuilder<Number, Set<T>>() {
+			@Override
+			public IType<? super Number> getValueType() {
+				return UnitLookup.RAW_NUMBER;
```

Comment:
If we're going this route, I guess this should be RAW_LONG. It does make process identifiers prone to any changes to default formatting of longs though. Another solution would be to introduce a ProcessIdentifier content type, and handle the formatting of that content type more explicitly. This is a smaller change though, and I think acceptable for now.

Reply:
I have changed the RAW_NUMBER to RAW_LONG in aggregator. 

But JDKAttribute will still be RAW_NUMBER because pid content type is in raw number format only.

<img width="380" alt="ContentTypePID" src="https://github.com/openjdk/jmc/assets/11155712/95568df7-08f5-4ea6-823a-89cd85d0f9d7">



 If I change it there then accessors are not compatible and the attribute value is not read. It shows N/A on JVM Internal screen.
<img width="419" alt="image" src="https://github.com/openjdk/jmc/assets/11155712/138c684f-c3cf-4908-9292-57d8fab1a279">


 However it shows proper value in Event Browser and Properties tab.
<img width="960" alt="image" src="https://github.com/openjdk/jmc/assets/11155712/0661f352-7155-45e9-8412-1db031fff0ee">